### ISSUE TITLE
Request: Support Old Tags

### ISSUE THREAD

**Issue by ghost** [ikrishg/reseter.css#38]
**Is your feature request related to a problem? Please describe.**
Support tags like `acronym`

**Describe the solution you'd like**
Use `abbr, acronym` instead of `abbr`.

**Comment by ghost** [comment]
This is a good first issue open for everyone. If you're interested kindly comment down below!

**Comment by verma-kunal** [comment]
Hey @krishdevdb !! I wanna contribute to this.
What do we have to do here, can u pls guide ?

**Comment by ghost** [comment]
@Zipher007 You first need to tell us that in which source will you be able to write the code (sass/scss)

**Comment by verma-kunal** [comment]
Ohk! Honestly speaking I am not familiar with either of those !
I have experience with css, but not worked with sass or scss !! Will have to learn that first, I suppose😕

**Comment by verma-kunal** [comment]
you can probably guide me here !!

**Comment by ghost** [comment]
Scss works just like css. We are not using any of the crazy features. So it will be easy for you.

Just change the `abbr` selector to select both `abbr, acronym`

**Comment by verma-kunal** [comment]
And precisely in which file(s) do we have to make these changes?
I am seeing a main `reseter.css` file and in the `src`, there are `sass`, `scss`, and other files 
(which ig are all connected with `reseter.css`)

**Comment by amazing-aryan-gupta** [comment]
@Zipher007 all files in the src are connected. You can start by editing `src/scss/reseter.scss`

**Comment by verma-kunal** [comment]
Alright ! So, what I have understood till now is:
Here is the original piece of code with `abbr` selector:
```
abbr[title],
abbr[data-bs-original-title] {
  text-decoration: underline dotted;
  cursor: help;
  text-decoration-skip-ink: none;
}
```
Now, we need to add another selector i.e `acronym`, with same attributes like `title` and `data-bs-original-title` like this:
```
abbr[title],
abbr[data-bs-original-title],
acronym[title],
acronym[data-bs-original-title] {
  text-decoration: underline dotted;
  cursor: help;
  text-decoration-skip-ink: none;
}
```
Is this the approach that we have to follow ?

**Comment by ghost** [comment]
```css
abbr[title],
abbr[data-bs-original-title],
acronym[title],
acronym[data-bs-original-title] {
  text-decoration: underline dotted;
  cursor: help;
  text-decoration-skip-ink: none;
}
```

Seems Perfect

**Comment by ghost** [comment]
After This, Whatever There is in the scss file should be copied to `src/less/reseter.less` (The syntax of less and scss is same). And I'll add the sass src once you are done with that and you open your pr.

**Comment by verma-kunal** [comment]
Alright !! Will do that

**Comment by verma-kunal** [comment]
Actually I was reading about these selectors at StackOverflow and this is what I found about `acronym`:
**Obsolete This feature is obsolete. Although it may still work in some browsers, its use is discouraged since it could be removed at any time. Try to avoid using it.**

What do you think about this ?

**Comment by ghost** [comment]
It depends on you whether you want to use it or not. We added it because, adding a single word to the file won't be something big. People can still use `abbr`. So it doesn't hurt adding `acronym`.

**Comment by verma-kunal** [comment]
Alright!! Understood
i've made the pull request !!

**Comment by Vikram-Hegde** [comment]
Hey there ! I would like to contribute to this project too. Let me know if I could help in anyway. (I'm a newbie too, this will be my first experience contributing)

**Comment by verma-kunal** [comment]
Hey @Vikram-Hegde !! Glad to have you here !!😃
Actually I've already tried to fix this bug and made a pull request, but surely you can stick around and @krishdevdb can help with you other contributions or some changes !!

**Comment by verma-kunal** [comment]
You can definitely check out the **`#good first issue`** channel on EddieHub discord to find more such beginner friendly issues !!

**Comment by Vikram-Hegde** [comment]
Thank you @Zipher007 will check it out !

**Comment by ghost** [comment]
Fixed in #39
